plugins/colorbutton/plugin.js

```diff
@@ -87,7 +87,8 @@ CKEDITOR.plugins.add( 'colorbutton', {
 			var style = new CKEDITOR.style( config[ 'colorButton_' + type + 'Style' ] ),
 				colorBoxId = CKEDITOR.tools.getNextId() + '_colorBox',
 				colorData = { type: type },
-				panelBlock;
+				panelBlock,
+				commandName = type === 'back' ? 'bgColor' : 'textColor';
```

Comment:
It seems like a command name could be also passed as an argument reducing magic done by function :thinking: 
Could you also refactor `addButton` like `renderColors` to accept options object instead of so may arguments?